ros2topic/ros2topic/api/__init__.py

```diff
@@ -177,11 +184,57 @@ def profile_configure_short_keys(
 
 
 def qos_profile_from_short_keys(
-    preset_profile: str, reliability: str = None, durability: str = None,
-    depth: Optional[int] = None, history: str = None,
+    preset_profile: str, reliability: Optional[str] = None, durability: Optional[str] = None,
+    depth: Optional[int] = None, history: Optional[str] = None, liveliness: Optional[str] = None,
+    liveliness_lease_duration_s: Optional[float] = None,
 ) -> rclpy.qos.QoSProfile:
     """Construct a QoSProfile given the name of a preset, and optional overrides."""
     # Build a QoS profile based on user-supplied arguments
     profile = rclpy.qos.QoSPresetProfiles.get_from_short_key(preset_profile)
-    profile_configure_short_keys(profile, reliability, durability, depth, history)
+    profile_configure_short_keys(
+        profile, reliability, durability, depth, history, liveliness, liveliness_lease_duration_s)
     return profile
+
+
+def add_qos_arguments(parser: ArgumentParser):
+    default_profile_str = 'sensor_data'
+
+    parser.add_argument(
+        '--qos-profile',
+        choices=rclpy.qos.QoSPresetProfiles.short_keys(),
+        help='Quality of service preset profile to subscribe with (default: {})'
+                .format(default_profile_str),
+        default=default_profile_str)
+    default_profile = rclpy.qos.QoSPresetProfiles.get_from_short_key(default_profile_str)
+    parser.add_argument(
+        '--qos-depth', metavar='N', type=int,
+        help='Queue size setting to subscribe with '
+                '(overrides depth value of --qos-profile option)')
```

Comment:
Nit : Minor flake8 errors on lines 206,212,217,223,229,235,240.

Reply:
fixed them in https://github.com/ros2/ros2cli/pull/788/commits/b341ee576b4687d8cba5b8e8b0bba5e66a3f8825